nilearn/tests/test_signal.py

```diff
@@ -366,7 +367,23 @@ def test_clean_confounds():
                                    np.zeros((20, 2)))
 
 
+def test_clean_img():
+
```

Comment:
We should always do that, because people copy-paste code to write their own test, and we want them to use this pattern.